next-cmake/host-library/CMakeLists.txt

```diff
@@ -0,0 +1,147 @@
+# Copyright Advanced Micro Devices, Inc., or its affiliates.
+# SPDX-License-Identifier: MIT
+
+set(TEMP_HOST_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../library)
+set(_CMAKE_CURRENT_SOURCE_DIR ${TEMP_HOST_SOURCE_DIR})
```

Comment:
The source tree could be device-library if that is a documented convention, but simpler is better and just use library/  as we only have one in rocblas